createmobilespec/README.md

```diff
@@ -78,6 +78,9 @@ If anything is going wrong and the existing output does not help, add the `--deb
 
 ### Create the App
 
+Caution: If the generated `mobilespec` project already exists, it is recommended
+to delete the project before running `createmobilespec.js`.
```

Comment:
```suggestion
to delete the project before proceeding.
```

Reply:
Done (with a force push)

Reply:
You can actually just include those in the GitHub UI, no need to do it manually - a lot faster. (Merging via Squash+Merge will make one commit out of it anyway)